consensus/snapshot/tests/test_state_delta.c

```diff
@@ -16,7 +16,7 @@ static char *snapshot_conf_path = "consensus/snapshot/tests/snapshot_conf.json";
 snapshot_t snapshot;
 iota_consensus_conf_t conf;
 
-void test_delta_serialization() {
+void test_snapshot_create_and_apply_patch() {
```

Comment:
Noooo don't change this name ! I think your IDE is changing in the whole repository, change only this one now :)